llvm/include/llvm/Frontend/HLSL/HLSLBinding.h

```diff
@@ -98,37 +99,52 @@ class BindingInfo {
 
   friend class BindingInfoBuilder;
 };
+struct Binding {
```

Comment:
Please put blank lines between top level definitions for readability and consistency.